src/components/NewSquad.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Paper, InputBase, Container, Typography, Button } from '@material-ui/core';
+import SquadsStyles from '../assets/jss/styles/NewSquad.styles.jsx';
+import { useMutation } from '@apollo/react-hooks';
+import SVG from 'react-inlinesvg';
+import Select from '@material-ui/core/Select';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import { SET_SQUAD } from '../requests';
+import { useSelector } from 'react-redux';
+
+
+const useStyles = makeStyles(SquadsStyles);
+
+export default function NewSquad() {
+  const classes = useStyles();
+  const user = useSelector(state => state.currentUser)
+  const [day, setDay] = React.useState(0);
+  const [squadNumber, setSquadNumber] = React.useState();
+  const [sendRequest] = useMutation(SET_SQUAD, { onCompleted: () => window.location.href ='/my-squad' });
+
+  if (user && user.squad) {
+    return window.location.href ='/my-squad'
+  }
```

Comment:
please do not use `window.location.reload` . As I remember I changed that in my commit here, incorrect conflict resolution?